src/gamestate/game_scene.cpp

```diff
@@ -919,11 +927,9 @@ void clean_up_basic_game_scene(sys::state& state) {
 }
 
 void update_army_group_selection_ui(sys::state& state) {
-	if(state.selected_army_group != nullptr) {
-		state.ui_state.army_group_window_sea->set_visible(state, true);
+	if(state.selected_army_group) {
```

Comment:
small elegance improvement: you can just replace `true` or `false` with this and turn the function into a one liner